gov2/iam/ListAdmins/ListAdminsv2.go

```diff
@@ -0,0 +1,289 @@
+		}
+	}
+
+	return false
+}
+
+// Documentation for func FUNCTION(c context.Context, api SERVICEMETHODAPI, input *SERVICE.METHODInput) (*SERVICE.METHODOutput, error)
+// FUNCTION VERBs an Amazon/AWS SERVICE RESOURCE.
+// Inputs:
+//     c is the context of the method call, which includes the AWS Region.
+//     api is the interface that defines the method call.
+//     input defines the input arguments to the service call.
+// Output:
+//     If success, a METHODOutput object containing the result of the service call and nil.
+//     Otherwise, nil and an error from the call to METHOD.
+
+func attachedUserPolicyHasAdmin(user *types.UserDetail, admin string) bool {
+	for _, policy := range user.AttachedManagedPolicies {
+		if *policy.PolicyName == admin {
+			return true
+		}
+	}
+
+	return false
+}
+
+// Documentation for func FUNCTION(c context.Context, api SERVICEMETHODAPI, input *SERVICE.METHODInput) (*SERVICE.METHODOutput, error)
+// FUNCTION VERBs an Amazon/AWS SERVICE RESOURCE.
+// Inputs:
+//     c is the context of the method call, which includes the AWS Region.
+//     api is the interface that defines the method call.
+//     input defines the input arguments to the service call.
+// Output:
+//     If success, a METHODOutput object containing the result of the service call and nil.
+//     Otherwise, nil and an error from the call to METHOD.
+
+func groupPolicyHasAdmin(c context.Context, client *iam.Client, group *types.Group, admin string) (bool, error) {
+	input := &iam.ListGroupPoliciesInput{
+		GroupName: group.GroupName,
+	}
+
+	result, err := client.ListGroupPolicies(c, input)
+	if err != nil {
+		return false, err
+	}
+
+	// Wade through policies
+	for _, policyName := range result.PolicyNames {
+		if *policyName == admin {
+			return true, nil
+		}
+	}
+
+	return false, nil
+}
+
+// Documentation for func FUNCTION(c context.Context, api SERVICEMETHODAPI, input *SERVICE.METHODInput) (*SERVICE.METHODOutput, error)
+// FUNCTION VERBs an Amazon/AWS SERVICE RESOURCE.
+// Inputs:
+//     c is the context of the method call, which includes the AWS Region.
+//     api is the interface that defines the method call.
+//     input defines the input arguments to the service call.
+// Output:
+//     If success, a METHODOutput object containing the result of the service call and nil.
+//     Otherwise, nil and an error from the call to METHOD.
+
+func attachedGroupPolicyHasAdmin(c context.Context, client *iam.Client, group *types.Group, admin string) (bool, error) {
+	input := &iam.ListAttachedGroupPoliciesInput{
+		GroupName: group.GroupName,
+	}
+
+	result, err := client.ListAttachedGroupPolicies(c, input)
+	if err != nil {
+		return false, err
+	}
+
+	for _, policy := range result.AttachedPolicies {
+		if *policy.PolicyName == admin {
+			return true, nil
+		}
+	}
+
+	return false, nil
+}
+
+// Documentation for func FUNCTION(c context.Context, api SERVICEMETHODAPI, input *SERVICE.METHODInput) (*SERVICE.METHODOutput, error)
```

Comment:
Template? Etc. throughout this file.

Reply:
Yep, fixed. Go functions that start with a lower-case letter are considered internal/private, so do not require documentation.